emily/handler/src/api/handlers/withdrawal.rs

```diff
@@ -408,28 +399,78 @@ async fn update_withdrawals(
         body.try_into_validated_update_request(api_state.chaintip().into())?;
 
     // Create aggregator.
-    let mut updated_withdrawals: Vec<(usize, Withdrawal)> =
+    let mut updated_withdrawals: Vec<(usize, WithdrawalWithStatus)> =
         Vec::with_capacity(validated_request.withdrawals.len());
 
     // Loop through all updates and execute.
     for (index, update) in validated_request.withdrawals {
+        if update.is_err() {
+            updated_withdrawals.push((
+                index,
+                WithdrawalWithStatus {
+                    withdrawal: Withdrawal::default(),
+                    status: StatusCode::BAD_REQUEST.as_u16(),
+                },
+            ));
+            continue;
+        }
+        let update = update.unwrap();
```

Comment:
Same stylistic comment here with using the `let .. else` syntax. No need to change though